wordpress-mobile/WordPress-Android
# Accessibility : Posting Activity Stats Improvement

Fixes #10975 

## Findings
The Posting Stats was inaccessible when using TalkBack. This improvement makes it more accessible for visually impaired users who are navigating the stats functionality. 

## Solution
The solution was to loop through each `Box` that represents a day and announce the associated stats. This implementation is similar to [what was done on iOS.](https://github.com/wordpress-mobile/WordPress-iOS/pull/13123)  The `Boxes` didn't have a content description so refactoring was done to generate it based on the month, day and amount of posts. 

Before | After 
--------|-------
[Before Video ](https://drive.google.com/open?id=18ErO8-vpZxLuxucriVecOq218nEcmaSK) |   [After Video    ](https://drive.google.com/open?id=186KK4cugtT2PX-SzyCCeGM8ka3UyzPKR)

## Testing

1. Enable TalkBack. 
2. Navigate to the Stats page.
3. Ensure you are on the Insights Tab. 
4. Select each Month under "Posting Activity" and follow the instructions to hear the announcement. 


Tap To Hear Day Stats | Stat Announced 
--------|-------
<img src="https://user-images.githubusercontent.com/1509205/71115552-2fe14d80-21a0-11ea-8fd9-427a58b41d14.png" width="320">        |       <img src="https://user-images.githubusercontent.com/1509205/71115581-3f609680-21a0-11ea-8947-9a264a142cd2.png" width="320">




All Stats Announced | No Stats Available 
--------|-------
<img src="https://user-images.githubusercontent.com/1509205/71115354-bea19a80-219f-11ea-9193-8bcec2daeb6c.png" width="320">   |    <img src="https://user-images.githubusercontent.com/1509205/71115439-e2fd7700-219f-11ea-8c70-34569e4c7c2e.png" width="320">


## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
